test/verification_script.js

```diff
@@ -0,0 +1,104 @@
+const TestToken = artifacts.require("DetailedMintableToken")
+const { getAllowances, assertNoAllowances } = require("../scripts/utils/trading_strategy_helpers")(web3, artifacts)
+
+contract("verification checks", async accounts => {
+  describe("allowances", async () => {
+    let tokenInfo
+
+    const createToken = async function(symbol, decimals) {
```

Comment:
You could lift this function to some utils file in the testing folder. Then it is easier to reuse